src/briefcase/integrations/docker.py

```diff
@@ -311,17 +363,19 @@ def run(self, args, env=None, **kwargs):
         # Any exceptions from running the process are *not* caught.
         # This ensures that "docker.run()" behaves as closely to
         # "subprocess.run()" as possible.
-        self._subprocess.run(
+        self.tools.logger.info("Entering Docker context...", prefix=self.app.app_name)
```

Comment:
These "enter" and "exit" print statements aren't near as clean as they were in the `dockerize` context manager (since they printed later and earlier (respectively) than they were previously) but they still seemed useful over all.